.github/workflows/lint-review.yaml

```diff
@@ -0,0 +1,25 @@
+name: autofix.ci
+
+on:
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  gofmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: "go.mod"
+
+      - run: gofmt -w -s .
+
+      - uses: autofix-ci/action@ea32e3a12414e6d3183163c3424a7d7a8631ad84
```

Comment:
What SHA is this?

Reply:
It's latest commit id of https://github.com/autofix-ci/action
You could see their official readme also suggest to use hash/commitId